Unbound/unbound/tests/domain.json

```
@@ -0,0 +1,37 @@
+{
+  "input": {
+    "message": "2019-10-08T11:29:05+02:00 hostfoo unbound[17265]: [17265:13] info: 127.0.0.1 reachms.corp.com. A IN"
+  },
+  "expected": {
+    "message": "2019-10-08T11:29:05+02:00 hostfoo unbound[17265]: [17265:13] info: 127.0.0.1 reachms.corp.com. A IN",
+    "dns": {
+      "question": {
+        "class": "IN",
+        "name": "reachms.corp.com.",
```

Can you see to remove the trailing dot in the domain-name?
```suggestion
        "name": "reachms.corp.com",
```

---

:ok_hand: